public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingGroup.tsx

```diff
@@ -90,86 +81,47 @@ function ConditionalRenderingGroupRenderer({ model }: SceneComponentProps<Condit
   const styles = useStyles2(getStyles);
   const { condition, value } = model.useState();
 
-  const conditionsOptions: Array<SelectableValue<'and' | 'or'>> = useMemo(
+  const conditionsOptions: Array<ComboboxOption<'and' | 'or'>> = useMemo(
     () => [
-      { label: t('dashboard.conditional-rendering.group.condition.meet-all', 'Meet all'), value: 'and' },
-      { label: t('dashboard.conditional-rendering.group.condition.meet-any', 'Meet any'), value: 'or' },
+      { label: t('dashboard.conditional-rendering.group.condition.meet-all', 'All conditions are met'), value: 'and' },
+      { label: t('dashboard.conditional-rendering.group.condition.meet-any', 'Any condition is met'), value: 'or' },
     ],
     []
   );
 
   return (
     <Stack direction="column">
-      <ConditionHeader title={model.title} onDelete={() => model.onDelete()} />
-      <Field label={t('dashboard.conditional-rendering.group.condition.label', 'Evaluate conditions')}>
-        <RadioButtonGroup
-          fullWidth
-          options={conditionsOptions}
-          value={condition}
-          onChange={(value) => model.changeCondition(value!)}
-        />
-      </Field>
-
-      <Divider spacing={1} />
+      {value.length > 1 ? (
+        <>
+          <Field
+            label={t('dashboard.conditional-rendering.group.condition.label', 'Apply if')}
+            className={styles.field}
+          >
+            <Combobox
+              options={conditionsOptions}
+              value={condition}
+              onChange={({ value }) => model.changeCondition(value!)}
```

Comment:
```suggestion
              onChange={({ value }) => model.changeCondition(value)}
```

Don't think we need the `!` here